### PR TITLE
docs(react-sdk): connection quality indicator

### PR DESCRIPTION
**Pull request by arnautov-anton**

### CODE REVIEW

File: packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const readableConnectionQuality =
+  SfuModels.ConnectionQuality[participant.connectionQuality];
+
+console.log(readableConnectionQuality); // -> EXCELLENT
+```
+
+### Connection quality indicator component
+
+You'll most likely be displaying this indicator component inside each participant box ((ParticipantView)[TODO: link]) within a call layout, you can learn more about `ParticipantViewUI` customizations in [this guide](./14-participant-view-customizations.mdx).
```
Comment (szuperaz) [inline packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:31]:
```suggestion
You'll most likely be displaying this indicator component inside each participant box ((ParticipantView)[TODO: link]) within a call layout, you can learn more about `ParticipantViewUI` customizations in [the participant view customizations guide](./14-participant-view-customizations.mdx).
```

**Review by szuperaz — APPROVED** [review]

File: packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+- poor
+- good
+- excellent
+
+In this guide you'll learn how to build and implement your own primitive connection quality indicator.
+
+### How to reach the state
+
+Each (`StreamVideoParticipant`)[TODO: link] has a `connectionQuality` property which by itself is just a number representation of values in a `SfuModels.ConnectionQuality` enum. To get readable format out of it we'll just pass the `connectionQuality` variable as a key to said enum:
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:18]:
```suggestion
Each (`StreamVideoParticipant`)[TODO: link] has a `connectionQuality` property which by itself is just a numeric representation of values in a `SfuModels.ConnectionQuality` enum. To get readable format out of it we'll just pass the `connectionQuality` variable as a key to said enum:
```

File: packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const ConnectionQualityIndicator = ({
+  participant,
+}: ParticipantViewUIProps) => {
+  const readableConnectionQuality =
+    SfuModels.ConnectionQuality[participant.connectionQuality];
+
+  return (
+    <span>
+      {'⭐️'.repeat(connectionQuality)} - {readableConnectionQuality}
```
[the remark this answers is not in the document]
Comment (arnautov-anton) [reply packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:47]: It should be, good catch, thanks!

File: packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx (hunk 1 above)
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:31]:
```suggestion
You'll most likely be displaying this indicator component inside each participant box ((ParticipantView)[TODO: link]) within a call layout. You can learn more about `ParticipantViewUI` customizations in [the participant view customizations guide](./14-participant-view-customizations.mdx).
```

File: packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    </span>
+  );
+};
+
+// pass it down to call layout components or directly to ParticipantView component
```
Comment (MartinCupela) [inline packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:52]:
```suggestion
// render ConnectionQualityIndicator directly in ParticipantView component
```

Comment (arnautov-anton) [reply packages/react-sdk/docusaurus/docs/React/06-ui-cookbook/10-connection-quality-indicator.mdx:52]: This comment was meant for the `CustomParticipantViewUI` which should be passed down either to our call layout components or directly to `ParticipantView` component utilised in your own call layout.

**Review by MartinCupela — APPROVED** [review]

**Pull request merged**